lib/client/group/mediator-topics/remove.js

```diff
@@ -0,0 +1,33 @@
+var CONSTANTS = require('../../../constants');
+
+/**
+ * Initialsing a subscriber for Removing groups.
```

Comment:
Comment doesn't match method. Do we need to have comments here? It provides very little to no value over method names.